src/c/core/session/read_access.c

```diff
@@ -183,7 +183,7 @@ inline void read_format_data(
 
                 if (uxr_deserialize_SampleIdentity(&temp_buffer, &sample_id))
                 {
-                    length = (uint16_t)(length - (temp_buffer.offset - offset));
+                    uint16_t request_length = (uint16_t)(length - (temp_buffer.offset - offset));
```

Comment:
Should we remove ` (void) length;` on line 156 since we are actually using the variable `length`?

Reply:
Removed!